template.yaml

```diff
@@ -2700,8 +2700,7 @@ Resources:
         - !Ref UserProfileTableWriteAccessPolicy
         - !Ref TxmaQueueSendPermissionPolicy
         - !Ref RedisParametersAccessPolicy
-        - !Ref OrchSessionTableReadAccessPolicy
-        - !Ref OrchSessionTableWriteAccessPolicy
+        - !Ref OrchSessionTableReadAndWriteAccessPolicy
```

Comment:
Better to keep reading and writing as separate policies (unless the limit of 20 policies is being hit)

Reply:
I think the granularity means that we can easily add or remove part of a policy, without risk of affecting a policy the lambda does need. Eg if we were to remove a handler's access to write, but needed it to keep reading, we could just remove write, rather than replacing read-write with read. The reason to use read-write was because we hit the maximum number of policies.

Reply:
Also, can you add this policy in a separate PR as a one-line change? Talking with Carly about my work, there's a small risk that the lambda could get deployed before it has this new policy on it, meaning the lambda will briefly fail (for a matter of minutes at most, but still). It's just safer to add the policy then start using it

Reply:
Sure - Ill add it on another PR and merge it first.

Reply:
Just realised if keeping the granularity no change needs to happen here. Will just remove.